src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int base;
+    private final int height;
+
+    public IsoscelesTrapezoid(String color, int base) {
+        super(color);
+        this.base = base;
+        this.height = (int) ((base * Math.sqrt(3) / 2));
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability.